raiden/transfer/node.py

```diff
@@ -872,22 +881,119 @@ def handle_state_change(chain_state: ChainState, state_change: StateChange) -> T
     return iteration
 
 
-def update_queues(iteration: TransitionResult):
+def is_transaction_successful(chain_state, transaction, state_change):
+    # These transactions are not made atomic through the WAL, they are sent
+    # exclusively through the external APIs.
```

Comment:
If i understand correctly, what you meant is:
`they are received exclusively from external APIs`. Correct?